frontend/assets/stylesheets/components/_guardian-header.scss

```diff
@@ -0,0 +1,127 @@
+/* ==========================================================================
+   Guardian Header
+   ========================================================================== */
+
+.guardian-header {
+    height: 74px;
+    @include mq($from: mobile-wider) {
+        height: 90px;
+    }
+    @include mq($from: mobileLandscape) {
+        height: 109px;
+    }
+    @include mq($from: tablet) {
+        height: 123px;
+    }
+    display: block;
+    background-color: $blue;
+    position: relative;
+    z-index: 3;
+}
+
+.guardian-header__inner {
+    @include clearfix();
+    padding: 6px 10px 0 10px;
+
+    @include mq($from: mobileLandscape) {
+        padding: 12px 20px 0 20px;
+    }
+}
+
+.guardian-header__logo {
+    float: right;
+    clear: both;
+}
+
+.guardian-header__logo__link {
```

Comment:
As far as I'm aware (double-check with cc: @SiAdcock), the convention with BEM is to have your block, and elements below it at any level are referenced relative to the block, e.g.:

```
.my-div
    .my-div__header
        .my-div__title
    .my-div__content
        .my-div__link
        .my-div__main-text
```

So, instead of `guardian-header__logo__link`, you'd use `guardian-header__logo_link` or `guardian-header__logo-link`. I believe the thinking behind this is that the nesting can get a bit out of control.

Obviously this isn't a technical blocker, these are the opinions of people on the internet 🐫.